Sources/Sentry/include/SentryTracer+Private.h

```diff
@@ -5,4 +5,10 @@ SentryTracer ()
 
 @property (nonatomic, strong) SentryHub *hub;
 
+/**
+ * We need an unchanging identifier to track concurrent tracers against a static profiler.
+ * @c SentryTracer.traceId can be changed by consumers so is unfit for this purpose.
+ */
+@property (nonatomic, strong) SentryId *concurrencyID;
```

Comment:
`h`: This has to be immutable.
```suggestion
@property (nonatomic, readonly, strong) SentryId *concurrencyID;
```

Reply:
I tried this and redeclaring it as readwrite in SentryTracer.m and it failed with error: duplicate declaration

Reply:
nm, don't even need the readwrite redeclare, just have to set the ivar in init